src/Webroot.php

```diff
@@ -138,12 +138,15 @@ public function putdb_PostgreSQLDatabase($conf, $sspak, $dropdb) {
 	public function putassets($sspak) {
 		$details = $this->details();
 		$assetsPath = $details['assets_path'];
-		$assetsOldPath = $assetsPath . '.old';
+		$assetsPath = escapeshellarg($assetsPath);
+
+		// Check for symlink - this was more reliable than is_link
+		$assetsPath = $this->exec("if [ -L {$assetsPath} ]; then readlink -f {$assetsPath}; else echo {$assetsPath}; fi")["output"];
```

Comment:
I'm really cautious about running shell args - I take it a pure PHP approach didn't work?

Reply:
Please quote the vars:

`"if [ -L \"{$assetsPath}\" ]; then readlink -f \"{$assetsPath}\"; else echo \"{$assetsPath}\"; fi"`

Reply:
Does this syntax work on php 5.3 machines?

Reply:
PHP didn't work, no - `is_link` returned false on my setup, I presume because it's fake-Windows so it isn't a link by PHP standards? Not really sure how exactly it figures it out but it gave me the wrong result.

Will pull the array out to help stragglers on PHP 5.3

And `escapeshellarg` should handle the quoting of the string :)